### ISSUE TITLE
Visualiser Date "Bill of Loading" dans "Echéances" de l'onglet LETTRAGE

### ISSUE THREAD

**Issue by lgazeau** [lgi-wines/Extranet-LGI#207]
Besoin de visualiser Date "Bill of Loading" dans "Echéances" de l'onglet LETTRAGE
Ex :; ClientID 143 WINESELLERS

KartID 39029 devrait remonter Date "Bill of Loading" 10/06/2018
KartID 38911 devrait remonter Date "Bill of Loading" 01/06/2018

**Comment by Portekoi** [comment]
C'est dans ce tableau qu'il faut ajouter la colonne, c'est bien ça? 

![image](https://user-images.githubusercontent.com/1560754/45176752-46fd6580-b211-11e8-96b9-4933a87e9407.png)

**Comment by lgazeau** [comment]
Oui, dans celui là et dans celui qui est au dessus (toujours sur la même page) ; celui qui récapitule les échéances.
